### PR TITLE
Create about

### PR DESCRIPTION
**Pull request by efronnik**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .header {
   background-color: aqua;
 }
+
+.abaut {
```
Comment (StepanovaOk) [inline css/styles.css]: Mistake class name

File: css/styles.css [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+  background-kolor: #212121;
```
Comment (StepanovaOk) [inline css/styles.css]: Mistake in word Color

File: index.html [hunk 3]
```diff
@@ -1,18 +1,30 @@
...
+<body>
+  <header class="header">
+    <p>
+      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ratione
+      excepturi, commodi harum molestiae odit dolores totam. Recusandae, eaque
+      ab reiciendis numquam necessitatibus voluptatibus iusto rem, unde
+      suscipit nam magnam quo.
+    </p>
+  </header>
+  <section class="abaut">
```
Comment (StepanovaOk) [inline index.html]: Mistake class name

File: css/styles.css (hunk 1 above)
Comment (efronnik) [reply css/styles.css]: fixed

File: css/styles.css (hunk 2 above)
Comment (efronnik) [reply css/styles.css]: fixed

File: index.html (hunk 3 above)
Comment (efronnik) [reply index.html]: fixed

**Review by StepanovaOk — APPROVED** [review]
It`s ok

**Pull request merged**